torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -1171,17 +1234,22 @@ def call_triton_kernel(
         from triton import JITFunction
         from triton.runtime.autotuner import autotune, Autotuner, Config, Heuristics
 
-        SPECIAL_CONFIG_NAMES = {"num_warps", "num_stages", "num_ctas"}
-
+        # Check if num_ctas is in kwargs
         if "num_ctas" in kwargs:
             self.raise_unsupported(
                 "Passing num_ctas directly to the Triton kernel is not supported. "
                 "Please use a Config in @triton.autotune instead."
             )
 
-        # Currently, if there are multiple autotuning decorators, the subsequent ones will be silently ignored.
-        # We also don't support the @triton.heuristics wrapper yet.
-        # We raise an error here to avoid silent incorrectness in these cases
+        # Make sure the kernel has a grid
+        if variable.grid is None:
+            self.raise_unsupported("Triton kernels should always be called with a grid")
+
+        """
+        Currently, if there are multiple autotuning decorators, the subsequent ones will be silently ignored.
```

Comment:
Please remove this comment as confusing. This is already addressed in https://github.com/pytorch/pytorch/pull/143519.